languages_substitution.go

```diff
@@ -100,3 +100,26 @@ var trSub = map[rune]string{
 	'ç': "c",
 	'Ç': "C",
 }
+
+var kkSub = map[rune]string{
+	'&': "jane",
+	'ә': "a",
```

Comment:
You missed mapping `@` symbol.

Reply:
People don't use `@` at Kazakh, that's why I used default `@`=>`` ( empty )

Reply:
This is strange for me. What about writing email address in text (like `mail@example.com`)?

Personally I would prefer handling `@` but will merge it as is for now, and fix if someone complain.